### DISCUSSION TITLE
Ideas for potential re-design, marketing chart

### DISCUSSION THREAD

**Discussion by hugolgst** [olivia-ai/olivia#160] [Ideas]
Hi everyone, 

I am planning on re-designing Olivia's brand, the colours, the fonts, logo etc.
Would you have any idea to make it better, any kind of feedback?

**Comment by YusufSuleman** [comment]
Use a VERY light grey in the background

**Reply by hugolgst** [reply]
Thanks for the suggestion! Keeping that in mind.
